build/template-OneBranch-Release-libsandsamples.yaml

```diff
@@ -0,0 +1,20 @@
+# template-OneBranch-Release-libsandsamples.yaml
+# Performs Nuget Restore and Build of LibsAndSamples.sln based on BuildPlatform and BuildConfiguration
+# This also includes a pack and sign of the MSAL.NET packages
+
+parameters:
+  BuildPlatform: 'any cpu'
+  BuildConfiguration: 'release'
+  MsalClientSemVer: '4.0.0-devopsbuild'
```

Comment:
Pls use smth like `4.55.0-devopsbuild`, because there is ESTS logic that is blocked on MSAL version and `4.0.0` is too old, i.e. tests will actually behave differently.